tasks/image-guest.yml

```diff
@@ -0,0 +1,103 @@
+---
+
+# Most of this code will run on the hypervisor as defined by the hyper varible
+# We use a block to reduce repeating the 'delegate_to' for every task
+- block:
+
+  - name: Ensure guestfish is installed
```

Comment:
For this role to work with a non-root user this task needs `become: yes` added to it. Otherwise an error will be produced because a non-root user cannot install packages.